features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/modules/statistics/usage.jag

```diff
@@ -18,7 +25,7 @@ function getFirstAccessTime(providerName, server) {
             };
 
         } else {
-
+            usage=JSON.parse(gson.toJson(usage));
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/9619da2271b49f37ef93456b6f8a8ac235f685ea